specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-06-01/sqlPool.json

```diff
@@ -693,6 +693,12 @@
             },
             "description": "OK"
           },
+          "201": {
+            "schema": {
+              "$ref": "#/definitions/SqlPool"
+            },
+            "description": "Accepted"
+          },
```

Comment:
Question on this one: I see that the operation can return a 202. Does is _sometimes_ return 202 and other times complete synchronously and return either 200 or 201?